internal/server/iigointernal/orchestration.go

```diff
@@ -0,0 +1,144 @@
+package iigointernal
+
+import (
+	"github.com/SOMAS2020/SOMAS2020/internal/common/baseclient"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/gamestate"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/roles"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/rules"
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"github.com/pkg/errors"
+)
+
+// featureJudge is an instantiation of the Judge interface
+// with both the Base Judge features and a reference to client judges
+var featureJudge = BaseJudge{
```

Comment:
not a fan of these package level globals.

Cue Tom Clarke `mutables are evil`

Well we still need mutables, but let's instead pass these around. These should be in the `GameState`, passed in an `IIGOInfo` struct variable perhaps. This struct can then be passed around and mutated. 

I strongly prefer returning mutated values and overwriting the old value, e.g.

```
j := s.gameState.IIGOInfo.Judge 
// do whatever u want with j, or even replace it
s.gameState.IIGOInfo.Judge = j
```